tests/integration_tests/test_integration_dds.py

```diff
@@ -434,6 +437,69 @@ def test_can_delivery_data_asynchronously(self):
+            self._create_projects_dir_with_random_data(tmpdir1, 'XYZ_123', os.path.basename(tmpdir1))
+            self._create_projects_dir_with_random_data(tmpdir2, 'XYZ_123', os.path.basename(tmpdir2))
+            
+            url = "/".join([self.API_BASE, "stage", "project", 'runfolders', 'XYZ_123'])
+            exclude_runfolder = [os.path.basename(tmpdir2)]
+            payload = {'delivery_mode': 'BATCH', 'exclude': json.dumps(exclude_runfolder)}
+            response = yield self.http_client.fetch(self.get_url(url), method='POST', body=json.dumps(payload))
+
+
+            response_json = json.loads(response.body)
+
+            staging_order_project_and_id = response_json.get("staging_order_ids")
+            staging_status_links = response_json.get("staging_order_links")
+            staged_data = response_json.get("staged_data")
+            # assert one runfolder was exluded from staging
+            self.assertEqual(len(staged_data), 1)
+            
+            for project, link in staging_status_links.items():
+                staging_id = staging_order_project_and_id[project]
+                while True:
+                    # Insert a pause to allow staging to complete
+                    time.sleep(1)
+                    status_response = yield self.http_client.fetch(link)
+                    if json.loads(status_response.body)["status"] == \
+                            StagingStatus.staging_successful.name:
+                        break
+
+                delivery_url = '/'.join([
+                    self.API_BASE, 'deliver', 'stage_id', str(staging_id)])
+                delivery_body = {
+                        'delivery_project_id': 'snpseq00025',
+                        'ngi_project_name': 'XYZ_123',
+                        'dds': True,
+                        'auth_token': '123',
+                        'skip_delivery': False,
+                        }
+
+                delivery_resp = yield self.http_client.fetch(
+                        self.get_url(delivery_url),
+                        method='POST',
+                        body=json.dumps(delivery_body))
+
+                delivery_resp_as_json = json.loads(delivery_resp.body)
+                delivery_link = delivery_resp_as_json['delivery_order_link']
+
+                while True:
+                    # Insert a pause to allow delivery to complete
+                    time.sleep(1)
+                    status_response = yield self.http_client.fetch(
+                            delivery_link)
+                    if (json.loads(status_response.body)["status"]
+                            == DeliveryStatus.delivery_successful.name):
+                        break
+                    elif (json.loads(status_response.body)["status"]
+                            == DeliveryStatus.delivery_failed.name):
+                        raise Exception("Delivery failed")
```

Comment:
I'm not sure the second half is necessary :thinking: :

* It makes this test very long
* Delivery is not included in any of the other tests that check different staging parameters (i.e. `test_can_stage_and_deliver_clean_flowcells`, `test_can_stage_and_deliver_batch_flowcells`,  `test_can_stage_and_deliver_force_flowcells` and `test_can_organise_stage_and_deliver_force_flowcells` only focus on staging).
* There are other tests such as `test_can_stage_and_delivery_runfolder` that check the combination of staging and delivery. I think that should be enough.